src/slice_box.rs

```diff
@@ -41,38 +43,42 @@ impl<T> typeob::PyTypeInfo for SliceBox<T> {
     }
 }
 
-impl<T: TypeNum> typeob::PyTypeCreate for SliceBox<T> {
+impl<T: TypeNum> type_object::PyTypeObject for SliceBox<T>
```

Comment:
Let's remove this `impl` block and use default implementation.
To use default impl, we have to add 
```rust
impl<T> PyMethodsProtocol for SliceBox<T> {
    fn py_methods() -> Vec<&'static PyMethodDefType> {
        Vec::new()
    }
}
```
(SliceBox is just a container which holds a pointer to Vec/ndarray and has no method